index.bs

```diff
@@ -1367,6 +1378,9 @@ enum XREye {
   readonly attribute XREye eye;
   readonly attribute Float32Array projectionMatrix;
   [SameObject] readonly attribute XRRigidTransform transform;
+  readonly attribute double? recommendedViewportScale;
+
+  undefined requestViewportScale(double? scale);
```

Comment:
```suggestion
  void requestViewportScale(double? scale);
```

Reply:
Done. I was following https://heycam.github.io/webidl/#idl-undefined which uses `undefined` in [examples](https://heycam.github.io/webidl/#create-sequence-from-iterable) and has this note:

> Note: This value was previously spelled void, and more limited in how it was allowed to be used.

Reply:
No, the use of `undefined` here is correct @Manishearth. We recently [merged that change](https://github.com/immersive-web/webxr/pull/1116) for the rest of the spec.